capstone/fitness/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ initIndex()
+ Function that runs when the body of index loads.
+ Actions:
+   - Check if the user is logged in and display their name or give them the 
+     option to log in. 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function initIndex() {
+  console.log("this works");
+  displayLogIn();
+}
+
+async function displayLogIn() {
+  const loginResponse = await fetch('/login');
+  const loginResponseText = await loginResponse.text();
+  var loginInfo = loginResponseText.split("\n");
+  
+  // This contains wheter the user is logged in, the user email, and login/out link
```

Comment:
nit: I think we can probably remove this comment since the code is self-explanatory.